packages/mantine/src/form/use-form.ts

```diff
@@ -0,0 +1,31 @@
+import {useForm as useMantineForm} from '@mantine/form';
+import {_TransformValues, UseFormInput, GetInputProps} from '@mantine/form/lib/types';
+
+export const useForm = <
+    Values = Record<string, unknown>,
+    TransformValues extends _TransformValues<Values> = (values: Values) => Values
+>(
+    options: UseFormInput<Values, TransformValues> = {}
+) => {
```

Comment:
Does the following works?

```suggestion
export const useForm: typeof useMantineForm = (options) => {
```

Reply:
With a small tweek, it does :+1: